<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>College Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBoxColleges">
      <property name="toolTip">
       <string>Select a College</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidgetDistances">
      <property name="toolTip">
       <string>Distance List</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidgetSouvenirs"/>
    </item>
    <item>
     <widget class="QListView" name="cartView"/>
    </item>
    <item>
     <widget class="QPushButton" name="lockButton">
      <property name="text">
       <string>Lock List</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelTotalDistance">
      <property name="text">
       <string>Total Trip Distance: 0 miles</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="maintenanceButton">
      <property name="minimumSize">
       <size>
        <width>839</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>false</bold>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Maintenance Mode</string>
      </property>
      <property name="iconSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
